editor/editor_properties.cpp

```diff
@@ -90,6 +90,10 @@ void EditorPropertyText::_text_changed(const String &p_string) {
 		return;
 	}
 
+	// Set tooltip so that the full text is displayed in a tooltip if hovered.
+	// This is useful when using a narrow inspector, as the text can be trimmed otherwise.
+	text->set_tooltip_text(text->get_text());
```

Comment:
Perhaps all of these plus the comment explaining it could be clumped up into a single inline function.

Reply:
This, or just remove all other comments. It doesn't need to be commented everywhere.

Reply:
I've removed most comments (now it's there only once per class).

Reply:
Now it seems even more like it needs a dedicated function. Though it would need to be shared by 2 classes. Maybe a macro?